packages/components/src/components/header/header.styles.ts

```diff
@@ -107,11 +107,11 @@ export default css`
   }
 
   .header__burger-menu-toggle:hover {
-    color: var(--syn-color-primary-600);
+    color: var(--syn-interactive-quiet-color-hover);
```

Comment:
we need the fallback token here:

```suggestion
    color: var(--syn-interactive-quiet-color-hover, var(--syn-color-primary-600));
```